### DISCUSSION TITLE
Is it possible to rebuild sidecars/thumbs from raw after xmp change?

### DISCUSSION THREAD

**Discussion by lansio** [photoprism/photoprism#2535] [Q&A]
Hello, step by step situation:
1. some CR3+XMP(produced by darktable4) were imported (with darktable-cli raw processor) sucessfully
2. next I put some fixes in xmp file
3. trying to rescan (regular or complete) same cr3 plus updated xmp

but photoprism even not trying to rebuild sidecar jpeg from cr3+ updated xmp :(

is it possible somehow force update/reimport/rebuild sidecars based on updated xmp?

thanks in advance

**Comment by lastzero** [comment]
For now, you have to update JPEGS manually when changing XMP values that affect appearance:

https://docs.photoprism.app/user-guide/faq/#library-counts-files

**Comment by lansio** [comment]
Oh, thank you so much for quick reaction!!!
I have seen this article, but it is unclear how can I perform the update "manually"? should I manually create a JPG (e.g. in darktable) and just put it in the storage section? or is any way to manually mark (somehow) the file for jpg-rebuilding in a normal process?

**Reply by lastzero** [reply]
Yes, same directory and name with jpg extension. Likely won't get automatically selected as primary. You can also configure photoprism in a way that puts created jpegs next to the originals so you can easily replace them. If you have an idea how a raw can be marked for refresh, let us know.

**Reply by lansio** [reply]
Thanks a lot for explanations!
